apps/admin/tailwind.config.ts

```diff
@@ -1,5 +1,6 @@
+// TODO: Why not use ESM in Tailwind config files?
+/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
[Question] I'm not super familiar with Tailwind configs, was this an intentional decision? I haven't tried it out locally but can if it wasn't.

Reply:
this might be a pretty old config but looks like the other tailwind configs arent disabling this rule

https://github.com/typehero/typehero/blob/main/apps/web/tailwind.config.ts

Reply:
I dug around a bit and maybe it wasn't supported yet at the time, but is now? https://tailwindcss.com/blog/tailwindcss-v3-3#esm-and-typescript-support

I tried an ESM import out in `apps/aot/tailwind.config.ts` and it worked nicely. So, hooray! Moving them all over. 🚀